src/test/java/gov/usgs/owi/nldi/transform/CharacteristicDataTransformerTest.java

```diff
@@ -44,6 +44,20 @@ public void initJsonTest() {
 		}
 	}
 
+	@Test
+	public void initJsonTestBad() {
+		try {
+			transformer.initJson(transformer.g, null);
+			transformer.g.flush();
+			assertEquals(INIT_JSON, response.getContentAsString());
+			fail("should have thrown runtime exception");
+		} catch (RuntimeException e) {
+			//ok
```

Comment:
Maybe being too nit picky here. Better to have a positive assertion of success rather than default:
boolean caughtRuntimeEx= false;
try {
...
}  (RuntimeException e) {
   caughtRuntimeEx = true;
}
assertTrue(caughtRuntimeEx)